src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -109,16 +104,17 @@ internal static Encoding GetDefaultEncoding()
 
         /// <summary>
         /// Facade for getting OEM encoding
+        /// OEM encodings work on all platforms, or rather codepage 437 is available on both Windows 
```

Comment:
Probably the comment intended to be ` codepage 437 is available on both Windows and Non-Windows`

Reply:
indeed so, fixed